src/isAnnotatedForRemoval.js

```diff
@@ -0,0 +1,7 @@
+// @flow weak
+
+export default function (node) {
+  const comments = node.trailingComments || [];
```

Comment:
Fine, I won't use it anyway, I would rather have the users decide of the syntax 👍 .